DEV-GUIDE.md

```diff
@@ -38,16 +44,16 @@ We have to verify 3 things here:
 ## Pull Request Convention
 
 - For the title, use imperative mood and capitalize it.
-- Use **rebase only** (No merge or squash).
+- Use **rebase only**.
 - **NO MERGE COMMITS**.
 - Push your works on your own forked repo, and make a **PR across forks**.
 - Use the same title (head) and content (body) for a single-commit PR.
 - When the PR resolves some issues, put `fixes #123` in the content of the PR, not in the commit messages.
 - Conversation **must be resolved by the commentor**, not the PR author.
-- Do not take the Github notifications as the communication channel; use Discord.
-- To resolve reviews, always put additional commits representing the changes, not amend the existing commits and force push it.
+- Do not take the Github notifications as the communication channel; use Discord and directly mention the relevant people.
```

Comment:
I was curious with this from the past actually.

> Do not take the Github notifications as the communication channel; use Discord
Does this sentence mean not to use Github comments to do discussions? but to use Discord?

Reply:
Clarified the sentence.